sdk/src/types/block/core/basic.rs

```diff
@@ -139,6 +139,24 @@ impl BasicBlock {
     pub fn burned_mana(&self) -> u64 {
         self.burned_mana
     }
+
+    /// Returns the work score of a [`BasicBlock`].
+    pub fn workscore(&self, workscore_structure: WorkScoreStructure) -> u32 {
```

Comment:
Where are these calculations from, so I can verify?

Reply:
iota-core

Reply:
We should ideally not look at iota-core. Is there not a TIP for this?

Reply:
no, unfortunately not yet

Reply:
The TIPs have been updated.